buildpipeline/pipeline.json

```diff
@@ -335,6 +348,19 @@
             "Type": "build/product/",
             "ConfigurationGroup": "Debug"
           }
+        }, 
+        {
+          "Name": "DotNet-CoreFx-Trusted-Linux-Native",
+          "Parameters": {
+            "DockerTag": "ubuntu1404_prereqs_v3",
+            "portableLinux": "-portableLinux"
```

Comment:
Do I understand it correctly that we perform the portable linux build on ubuntu 14.04? If that's the case, we need to change it to use CentOS 7.1 instead. The point is to build on Linux distro with the oldest libraries, especially libicu.